src/testrel.jl

```diff
@@ -20,14 +16,20 @@ function gen_safe_name(basename)
     return name[1:min(sizeof(name), 63)]
 end
 
-TEST_CONTEXT_WRAPPER::ContextWrapper = ContextWrapper(Context(load_config()))
+const TEST_CONTEXT = Ref{Option{Context}}(nothing)
 
-function get_context()::Context
-    return TEST_CONTEXT_WRAPPER.context
+try
+    TEST_CONTEXT[] = Context(load_config())
+catch
+    @warn "No `default` RAI context found. Use `set_context` to pass in a context and enable usage of `RAITest`."
+end
```

Comment:
i think we need to do this in `__init__` (i.e. at package _load_ time), otherwise this value will be baked in at _precompile_ time.

e.g. if i don't have a `default` config when i first precompile the package, then i go add a `default` config, then we want `TEXT_CONTEXT` to pick up that new `default` config when the package is next loaded, rather than having to `] precompile` to get the change

(style-wise, i'd normally define `__init__` in the main module file, `src/RAITest.jl`, to reduce the chance of multiple `__init__` functons being defined, but up to you)

```suggestion
function __init__()
    try
        TEST_CONTEXT[] = Context(load_config())
    catch
        @warn "No `default` RAI context found. Use `set_context` to pass in a context and enable usage of `RAITest`."
    end
end
```